rpcd/playbooks/roles/rpc_maas/files/plugins/disk_status.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python
```

Comment:
Oof, dumb error on my part. ``failing_disks`` should be set to 0 since HP can't show predictive failures easily.